corehq/apps/accounting/tests/test_subscription_changes.py

```diff
@@ -145,6 +146,46 @@ def test_resubscription(self):
         self._assertInitialRoles()
         self._assertStdUsers()
 
+    @patch('corehq.apps.toggle_ui.views.ATTENDANCE_TRACKING.enabled')
```

Comment:
Just FYI, there's also a @flag_enabled decorator you can use instead of patching. See example [here](https://github.com/dimagi/commcare-hq/blob/79e02d022d0d17765b5d454d15bfbaf1e7347ba3/corehq/apps/api/odata/tests/test_feed.py#L24).